tests/custom_lookups/tests.py

```diff
@@ -316,7 +345,7 @@ def test_lookups_caching(self):
         field = Article._meta.get_field("author")
 
         # clear and re-cache
-        field.get_lookups.cache_clear()
+        field.get_class_lookups.cache_clear()
```

Comment:
Do we need to switch to `get_class_lookups`?

Reply:
Here, as `field` is an instance, `get_lookups` redirects to `get_instance_lookups` doesn't have a `clear_cache` method (only `get_class_lookups` has). So clearing cache can only be done by `get_class_lookups`(which was previously called `get_lookups`).